includes/admin.form.inc

```diff
@@ -41,7 +41,12 @@ function islandora_multi_importer_admin_form($form, $form_state) {
     '#description' => t('This option also requires corresponding Drupal permissions to be correctly set.'),
     '#default_value' => variable_get('islandora_multi_importer_extrads', FALSE),
   );
-
+  $form['islandora_multi_importer_generalsettings_fieldset']['islandora_multi_importer_temp_dir'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Temporary files storage directory'),
+    '#description' => t('Path to where to store temporary files under Drupal\'s <em>private://</em> directory. This is used for storing batch ingest files, generated metadata files, and other temporary files. Do not include leading or trailing slash. The <em>private://</em> directory location is configured at <a href="/admin/config/media/file-system">Configuration > Media > File system</a>' ),
```

Comment:
Pat! I think i have a minor change need here. Do you think we should make the streamwrapper itself configurable? Select input?, not only the directory.  Reasons (good ones) are many, like when you have a cron task running and that deals with temporary:// uploaded files, or even custom ones, so you could want the temp stuff to go there instead of into private? Your new functions already allow that argument to be passed around. What do you think?

Reply:
Sorry I've left this unresolved for so long. I agree with configurable streamwrapper. I'll try to do that soon,